lib/src/main/java/io/ably/lib/push/Storage.java

```diff
@@ -0,0 +1,16 @@
+package io.ably.lib.push;
+
+import java.lang.reflect.Field;
+
+public interface Storage {
```

Comment:
Also, I was curious, if current `Storage` interface class is meant for both `ephermal` or `permanant` storage?

Reply:
If it also works for epthermal storage, you can close this one too : ), I will approve right away

Reply:
Not sure if I 100% understand what do you mean.  I've tested it in this PR where I used hash map as a storage, which is ephemeral storage.